src/main/java/org/springframework/integration/kafka/listener/QueueingMessageListenerInvoker.java

```diff
@@ -78,14 +95,8 @@ public void enqueue(KafkaMessage message) {
 			// handle the case when the thread is interrupted while the adapter is still running
 			// retry adding the message to the queue until either we succeed, or the adapter is stopped
 			while (!added && this.running) {
-				try {
-					this.messages.put(message);
-					added = true;
-				}
-				catch (InterruptedException e) {
-					// we ignore the interruption signal if we are still running, but pass it on if we are stopped
-					wasInterruptedWhileRunning = true;
```

Comment:
Since we don't catch `InterruptedException` any more, `wasInterruptedWhileRunning` variable is redundant. Removing...